src/content/docs/en/guides/astro-db.mdx

```diff
@@ -54,18 +52,12 @@ export default defineDb({
 
 Data in Astro DB is stored using SQL tables. Tables structure your data into rows and columns, where columns enforce the type of each row value.
 
-Define your tables in your `db/config.ts` file by providing the structure of the data in your existing libSQL database, or the data you will collect in a new database.
-
-When you define a table, Astro will generate a TypeScript interface to query that table from your project. The result is full TypeScript support when you access your data with property autocompletion and type-checking.
+Define your tables in your `db/config.ts` file by providing the structure of the data in your existing libSQL database, or the data you will collect in a new database. This will allow Astro to generate a TypeScript interface to query that table from your project. The result is full TypeScript support when you access your data with property autocompletion and type-checking.
 
-To configure a database table, import and use the `defineTable()` and `column` utilities from `astro:db`.
+To configure a database table, import and use the `defineTable()` and `column` utilities from `astro:db`. Then, create a name (case-sensitive) for your table and define the type of data in each column.
```

Comment:
Small nit pick

```diff
- Then, create a name (case-sensitive) for your table and define the type of data in each column.
+ Then, define a name (case-sensitive) for your table and the type of data in each column.
```

Reply:
idk why its not letting me apply the changes from github, lets try this
```suggestion
To configure a database table, import and use the `defineTable()` and `column` utilities from `astro:db`. Then, define a name (case-sensitive) for your table and the type of data in each column.
```

Reply:
it didnt list you as the co-author... oops sorry abt that!

Reply:
No worries, I’m not super concerned about that haha